pop11_grammar_ebnf.txt

```diff
@@ -25,8 +29,8 @@ Definition ::=
 	'define' Word ('(' Varslist? ')' | Varslist)? ('->' Varslist)* ';' Sequence 'enddefine'
 
 Assignment ::=
-	Expression ('->' | '->>') (Variable | FunctionApplication)
-	| Expression '-->' Expression
+	NonEmptyExpression ('->' | '->>') (Variable | FunctionApplication)

```

Comment:
Incorrect. Pop-11 allows for the LHS of assignments to be empty.
```
 Setpop
 : 42;
 : vars x;
 : -> x;
 : x =>
 ** 42
 : 
```
Also, the RHS of simple assignments may also be empty. Example:
```
 Setpop
 : 66 -> ;
 : 
```